src/plugins/yanlr/README.md

```diff
@@ -0,0 +1,117 @@
+- infos = Information about the yanlr plugin is in keys below
+- infos/author = René Schwaiger <sanssecours@me.com>
+- infos/licence = BSD
+- infos/needs = directoryvalue yamlsmith
```

Comment:
Does this work? (To use two different storage plugins at the same time)

Why the split? (Do you want to share yamlsmith across all parsers?)

Reply:
> Does this work?

Yes.

> Do you want to share yamlsmith across all parsers?

Yep, YAML Smith should act as serialization plugin for Yan LR and other upcoming YAML plugins.